fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/cascades/values/MessageHelpers.java

```diff
@@ -560,5 +561,10 @@ public CoercionTrieNode(@Nullable final BiFunction<Descriptors.Descriptor, Objec
         public CoercionTrieNode getThis() {
             return this;
         }
+
+        @Override
+        public int planHash(@Nonnull final PlanHashKind hashKind) {
+            return PlanHashable.objectPlanHash(hashKind, getChildrenMap());
```

Comment:
I don't think this is correct as the Map is not `PlanHashable` and will therefore delegate to the regular `hashCode()`.